h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -166,9 +218,9 @@ class SearchBarController extends Controller {
       true /*capture*/);
     this._input.addEventListener('focusout', handleFocusOutside,
       true /*capture*/);
-    this._input.addEventListener('input', handleFocusOnInput,
+    this._input.addEventListener('input', handleFocusinOnInput,
       true /*capture*/);
-    this._input.addEventListener('focusin', handleFocusOnInput,
+    this._input.addEventListener('focusin', handleInputOnInput,
```

Comment:
Looks like the wrong handler functions on the wrong events! (Which doesn't break anything since they both do the same thing)